c++tools/configure

```diff
@@ -2769,7 +2769,7 @@ if test "$ac_test_CXXFLAGS" = set; then
   CXXFLAGS=$ac_save_CXXFLAGS
 elif test $ac_cv_prog_cxx_g = yes; then
   if test "$GXX" = yes; then
-    CXXFLAGS="-g -O2"
+    CXXFLAGS="-g -Og"
```

Comment:
This and similar changes were added just to make debugging of the `Hello World` easier. We can revert it.